lib/garufa/subscriptions.rb

```diff
@@ -55,6 +56,11 @@ def channel_size(channel)
       (subscriptions[channel] || []).size
     end
 
+    def channel_stats(channel)
+      stats = API::Stats.new(subscriptions)
+      stats.single_channel(channel)
+    end
```

Comment:
Instead of creating a new instance of `Stats`, maybe we can have an instance cached for future uses.

I think of something like this:
```ruby
def stats
  @stats ||= API::Stats.new(subscriptions)
end

def channel_stats(channel)
  stats.single_channel(channel)
end
```

We can also take advantage of this change and use the `stats` instance to get the `channel_size` also:
```ruby
def channel_size(channel)
  stats.channel_size(channel)
end
```

Reply:
Great, I like the idea, will implement and test it that way and come back to this.